tsconfig.json

```diff
@@ -14,6 +14,11 @@
     "removeComments": true,
     "sourceMap": true,
     "strictNullChecks": true,
-    "target": "es6"
-  }
+    "target": "es6",
+    "baseUrl": ".",
+    "paths": {
+      "components/*": ["./src/components/*"]
+    }
```

Comment:
not sure if this is allowed, but should we have trailing commas in the tsconfig too?

Reply:
not technically allowed since that's malformed json, trailing commas should only be in js/ts/tsx files